src/model.ts

```diff
@@ -120,6 +120,8 @@ export class File {
     }
 
     public getAbsolutePath = (): string => {
+        if (this.path.startsWith('/') || this.path.startsWith('~'))
```

Comment:
Maybe "path.isAbsolute(myPath)" ? (path as in mode module). And 
if (isAbsolute) {
 return
}
else {
 return
}
because these two branches would be "a normal flow".